pkg/azure/azure.go

```diff
@@ -211,43 +211,41 @@ func EnsureImageDescriptions(
 		return fmt.Errorf("failed to obtain a credential: %w", err)
 	}
 
-	for _, location := range locations {
```

Comment:
azure resource, gallery and gallery image can be created with only one location. image can be replicated in multiple locations (regions). The replication of the images will be handled by packer.